test/server/cards/03_TWI/events/InPursuit.spec.ts

```diff
@@ -0,0 +1,52 @@
+describe('In Pursuit', function() {
+    integration(function(contextRef) {
+        describe('In Pursuit\'s ability', function() {
+            it('should exhaust an enemy unit if you exhaust a friendly unit', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['in-pursuit'],
+                        groundArena: ['battlefield-marine'],
+                        spaceArena: [{ card: 'green-squadron-awing', exhausted: true }]
+                    },
+                    player2: {
+                        groundArena: [{ card: 'pyke-sentinel', exhausted: true }],
+                        spaceArena: ['inferno-four#unforgetting']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.inPursuit);
+                expect(context.player1).toBeAbleToSelectExactly([context.battlefieldMarine]);
```

Comment:
As noted above, A-Wing is a legal target for this effect

Reply:
fixed up